packages/rangelink-vscode-extension/src/destinations/capabilities/insertFactories/terminalInsertFactory.ts

```
@@ -1,41 +1,35 @@
-import type { Logger } from 'barebone-logger';
 import type * as vscode from 'vscode';
 
-import type { ClipboardPreserver } from '../../../clipboard/ClipboardPreserver';
-import type { VscodeAdapter } from '../../../ide/vscode/VscodeAdapter';
-
 import type { InsertFactory } from './InsertFactory';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)